tests/foreman/cli/test_remoteexecution.py

```diff
@@ -911,7 +899,7 @@ def test_positive_run_scheduled_job_template(self, rex_contenthost, target_sat):
             invocation_info = target_sat.cli.JobInvocation.info({'id': invocation_command['id']})
             pending_state = invocation_info['pending']
             sleep(30)
-        assert_job_invocation_result(target_sat, invocation_command['id'], client.hostname)
+        assert invocation_info['status'] in ['running', 'finished', 'queued']
```

Comment:
Instead of using sleep func here, why not use wait_for here to check if job-invocation is `finished`?

Reply:
or `wait_for_tasks`.